python/ray/autoscaler/kuberay/kuberay-autoscaler.patch

```diff
@@ -1,13 +1,26 @@
 diff --git a/ray-operator/config/default/kustomization.yaml b/ray-operator/config/default/kustomization.yaml
-index 333106d..fdf62d6 100644
+index 7df72cd..b67c047 100644
 --- a/ray-operator/config/default/kustomization.yaml
 +++ b/ray-operator/config/default/kustomization.yaml
-@@ -21,6 +21,6 @@ bases:
- 
+@@ -23,5 +23,5 @@ bases:
  images:
  - name: kuberay/operator
--  newName: kuberay/operator
+   newName: kuberay/operator
 -  newTag: nightly
-+  newName: rayproject/kuberay-operator
-+  newTag: latest
++  newTag: a46ba3f
  
+diff --git a/ray-operator/config/manager/manager.yaml b/ray-operator/config/manager/manager.yaml
+index ef5318f..b5f32bf 100644
+--- a/ray-operator/config/manager/manager.yaml
++++ b/ray-operator/config/manager/manager.yaml
+@@ -21,8 +21,8 @@ spec:
+       containers:
+       - command:
+         - /manager
+-#        args:
+-#        - --enable-leader-election
++        args:
++        - --prioritize-workers-to-delete
```

Comment:
diffs of diffs are hard on my head --

What happened here? If I understand right, prioritize workers to delete is the feature flag on the KubeRay side.
What are the implications of removing --enabling-leader-election?

Reply:
Hey Dmitri, the --enabling-leader-election flag is not being removed, it was never active -- the `--enabling-leader-election` flag was commented out and the comment is removed :)

It is possible that we can do this better with kustomize.yaml, but I don't know much about it and the patch certainly works for now :) Happy to look a little more into whether we can do without the patch (and it is temporary anyways since the feature flag will be removed at some point).

Reply:
Got it. In my Saturday morning grog, I missed the `#` symbols.

Reply:
Once everything settles, we can recommend deploying using purely the files hosted in the KubeRay repo.
Patching works while autoscaling support is experimental.

Reply:
Yes, that sounds great :)